hw/dv/tools/Makefile

```diff
@@ -32,6 +32,7 @@ RUN_DIR           := ${RUN_PATH}/${RUN_LOC}
 COV_REPORT_DIR    ?= ${SCRATCH_PATH}/cov_report
 SW_BUILD_DIR      ?= ${RUN_DIR}/sw_build
 SW_ROOT_DIR       := ${PRJ_DIR}/sw
+SW_BUILD_SUFFIX   ?= fpga_nexysvideo
```

Comment:
I would consider naming this something like `SW_BUILD_DEVICE`, to be somewhat more descriptive.

Eventually, I expect this to change to `sim_dv` for you guys.

Reply:
good idea. Updated